apps/web/src/network/Network.tsx

```diff
@@ -226,6 +276,21 @@ export default function GraphNetwork() {
       _network.stopSimulation();
     });
 
+
+    _network.on('click', (params) => {
```

Comment:
It looks like `dfs` function (declared within `highlightDeepDependencies`) will be triggered on each click even though the `deep dependencies` option is not checked? I think it would be more convenient to trigger DFS only when the option is checked, I can imagine that on huge graphs computing such functions will have a cost we want to avoid most of the time

Reply:
Good point